src/dsp_tools/utils/logger_config.py

```diff
@@ -0,0 +1,45 @@
+import sys
+from datetime import datetime
+from pathlib import Path
+
+from loguru import logger
+
+logger_savepath = (Path.home() / ".dsp-tools" / "logger.log").absolute()
+
+
+def logger_config() -> None:
+    """
+    This function configures the log files.
+    Currently, there are three sinks:
+    - logger.log in the home directory for development purposes
+    - warnings.log in the cwd only with level warning and higher for the user (no stack-trace)
+    - print output on the terminal, formatted the same as the warnings.log
+    """
+    # If this is not removed, the default formatting is also printed out on the terminal
+    logger.remove()
+
+    text_format = "<level>{time:YYYY-MM:DD HH:mm:ss.SSS} | {level: <8} | {message}</level>"
+    rotation_size = "500 MB"
+    retention_number = 30
+    timestamp_str = datetime.now().strftime("%Y%m%d-%H%M")
+
+    logger.add(sink=sys.stdout, format=text_format, level="INFO", backtrace=False, diagnose=False)
```

Comment:
Cool. But shouldn't we now remove all the print statements which are just a duplication of what is already logged?

Reply:
True, we should decide either way. I spoke about it with balduin and he argues for not having any log statements printed out. I understand the argument, therefore in fact, we could remove this line and just keep the new logile

Reply:
To me, there is a fundamental difference between logging and user feedback: One is for a good user experience, the other is for easy diagnostics and tracing back what happened in exceptional cases. And for that reason I would like to keep them separate both conceptually and in code.  
But that is something we already have on our notion page, and that definitely should not block this PR; and if you like/feel the need, we can also discuss it again in our next meeting.

For now, I'm ok with either logging these messages, or printing them (the way it currently is set up, there is no conceptual difference anyways), but if they are being logged to stdout, they of course should not be printed, otherwise they show up twice.

Reply:
I think I will remove the sink here, because it is easier to add this than to re-introduce the print statements and then we can discuss this later.

Reply:
That sounds reasonable, then we can still discuss later how we want to proceed in the future